### ISSUE TITLE
A default value should be provided for IngressIPNetworkCIDR

### ISSUE THREAD

**Issue by marun** [openshift/origin#10387]
The assignment of ingress ip's is disabled by default.  Set IngressIPNetworkCIDR to a private range by default on bare metal to ensure that services of type loadbalancer will have ingress ip's assigned.

cc: @smarterclayton

**Comment by liggitt** [comment]
that would mean it would turn on for all old configs that are missing the field. is that what we want?

**Comment by liggitt** [comment]
is there a private IP range we can assume we own? I wouldn't have thought so

**Comment by marun** [comment]
@liggitt This was at the request of @smarterclayton.   He wanted a private range that didn't conflict with other CIDRs (ExternalIPNetworkCIDRs, ClusterNetworkCIDR and ServiceNetworkCIDR) set by default on bare metal to make it easier for users to play with the feature.

**Comment by smarterclayton** [comment]
There is already config that validates they don't overlap.  We don't *own* 172.16 / 30, but we can use them.  Similar concept.  Old clusters getting this I think is ok - it's a net new enablement.  We may want to have defaulting have a flag that checks if it overlaps with existing configured ranges and then does not default if it overlaps to avoid breaking old clusters.

```
defaulter:
  if externalIPNetworkCIDR == nil {
    cidr = "172.46.0.0/16" or something
    if cidr.Overlaps(clusterNetwork) || cidr.Overlaps(serviceNetwork) {
      cidr = nil
    }
  }
```

**Comment by marun** [comment]
Fixed by #10500
